ramls/item.json

```diff
@@ -296,6 +296,45 @@
       "folio:linkFromField": "holdingsRecordId",
       "folio:linkToField": "id",
       "folio:includedElement": "holdingsRecords.0"
+    },
+    "lastCheckIn": {
+      "type": "object",
+      "readonly": true,
+      "additionalProperties": false,
+      "properties": {
+        "date": {
```

Comment:
This appears to be a date and time. I'm aware that often those are named `date` rather than `date time` in FOLIO, that might be a topic to review at some point.

Could this definition be extended to include the appropriate format, to make it more explicit what is expected in this property.

Reply:
Done.